lhotse/recipes/timit.py

```diff
@@ -0,0 +1,295 @@
+        phonemes["jh"] = "jh"
+        phonemes["k"] = "k"
+        phonemes["kcl"] = "cl"
+        phonemes["l"] = "l"
+        phonemes["m"] = "m"
+        phonemes["n"] = "n"
+        phonemes["ng"] = "ng"
+        phonemes["nx"] = "n"
+        phonemes["ow"] = "ow"
+        phonemes["oy"] = "oy"
+        phonemes["p"] = "p"
+        phonemes["pau"] = "sil"
+        phonemes["pcl"] = "cl"
+        phonemes["q"] = ""
+        phonemes["r"] = "r"
+        phonemes["s"] = "s"
+        phonemes["sh"] = "sh"
+        phonemes["t"] = "t"
+        phonemes["tcl"] = "cl"
+        phonemes["th"] = "th"
+        phonemes["uh"] = "uh"
+        phonemes["uw"] = "uw"
+        phonemes["ux"] = "uw"
+        phonemes["v"] = "v"
+        phonemes["w"] = "w"
+        phonemes["y"] = "y"
+        phonemes["z"] = "z"
+        phonemes["zh"] = "zh"
+
+    elif num_phones == int(39):
+        print("Using 39 phones for modeling!")
+        # This dictionary is used to conver the 60 phoneme set into the 39 one.
+        phonemes["sil"] = "sil"
+        phonemes["aa"] = "aa"
+        phonemes["ae"] = "ae"
+        phonemes["ah"] = "ah"
+        phonemes["ao"] = "aa"
+        phonemes["aw"] = "aw"
+        phonemes["ax"] = "ah"
+        phonemes["ax-h"] = "ah"
+        phonemes["axr"] = "er"
+        phonemes["ay"] = "ay"
+        phonemes["b"] = "b"
+        phonemes["bcl"] = "sil"
+        phonemes["ch"] = "ch"
+        phonemes["d"] = "d"
+        phonemes["dcl"] = "sil"
+        phonemes["dh"] = "dh"
+        phonemes["dx"] = "dx"
+        phonemes["eh"] = "eh"
+        phonemes["el"] = "l"
+        phonemes["em"] = "m"
+        phonemes["en"] = "n"
+        phonemes["eng"] = "ng"
+        phonemes["epi"] = "sil"
+        phonemes["er"] = "er"
+        phonemes["ey"] = "ey"
+        phonemes["f"] = "f"
+        phonemes["g"] = "g"
+        phonemes["gcl"] = "sil"
+        phonemes["h#"] = "sil"
+        phonemes["hh"] = "hh"
+        phonemes["hv"] = "hh"
+        phonemes["ih"] = "ih"
+        phonemes["ix"] = "ih"
+        phonemes["iy"] = "iy"
+        phonemes["jh"] = "jh"
+        phonemes["k"] = "k"
+        phonemes["kcl"] = "sil"
+        phonemes["l"] = "l"
+        phonemes["m"] = "m"
+        phonemes["ng"] = "ng"
+        phonemes["n"] = "n"
+        phonemes["nx"] = "n"
+        phonemes["ow"] = "ow"
+        phonemes["oy"] = "oy"
+        phonemes["p"] = "p"
+        phonemes["pau"] = "sil"
+        phonemes["pcl"] = "sil"
+        phonemes["q"] = ""
+        phonemes["r"] = "r"
+        phonemes["s"] = "s"
+        phonemes["sh"] = "sh"
+        phonemes["t"] = "t"
+        phonemes["tcl"] = "sil"
+        phonemes["th"] = "th"
+        phonemes["uh"] = "uh"
+        phonemes["uw"] = "uw"
+        phonemes["ux"] = "uw"
+        phonemes["v"] = "v"
+        phonemes["w"] = "w"
+        phonemes["y"] = "y"
+        phonemes["z"] = "z"
+        phonemes["zh"] = "sh"
+    
+    else:
+        print("Using 60 phones for modeling!")
```

Comment:
Please be consistent: You're using both `logging` and `print`. Please choose either one, not both.